imports/ui/lists/recentAgents.coffee

```diff
@@ -1,72 +1,61 @@
 require './recentAgents.jade'
 
-pp = 75
+{ articles, agents } = require '../../data/collections.coffee'
 
-Template.recentAgents.onCreated ->
-  @recentAgents = new Meteor.Collection(null)
-  @articles = new Meteor.Collection(null)
-  @currentPageNumber = new ReactiveVar(0)
-  @isLoading = new ReactiveVar(false)
-  @theEnd = new ReactiveVar(false)
-  order = 0
-  @loadMoreArticles = =>
-    if @isLoading.get() then return
-    pageNum = @currentPageNumber.get()
-    @currentPageNumber.set(pageNum + 1)
-    # @recentAgents.find({}, reactive: false).map((d) => @recentAgents.remove(d))
-    @isLoading.set(true)
-    HTTP.get '/api/recentAgents', {params: {page: pageNum, pp: pp}}, (err, res) =>
-      @isLoading.set(false)
-      if err
-        toastr.error(err.message)        
-        return
-      unless res.data.results.length
-        @theEnd.set(true)
-        return
-      for row in res.data.results
-        articleId = @articles.findOne(uri: row.currentArticle)?._id
-        unless articleId
-          articleId = @articles.insert
-            uri: row.currentArticle
-            postSubject: row.postSubject
-            date: moment(new Date(row.currentDate))
-            collapsed: false
-            order: order++
-        row.articleId = articleId
-        if row.priorDate
-          row.priorDate = new Date(row.priorDate)
-          priorDate = moment(row.priorDate)
-          currentDate = moment(new Date(row.currentDate))
-          row.days = currentDate.diff(priorDate, 'days')
-          row.months = currentDate.diff(priorDate, 'months')
-          #show days or months since last mention
-          if row.days > 30
-            row.dm = true
-        @recentAgents.insert(row)
-      # ...
-      @articles.find().forEach (article) =>
-        if @recentAgents.find(articleId: article._id).count() > 5
-          @articles.update(article._id, { $set: { collapsed: true } })
+# Maximum amount of months to be visible at one given time
+maxVisibleMonths = 3
+
+skipMonths = new ReactiveVar(Number(sessionStorage.getItem 'skipMonths') or 0)
```

Comment:
That was my original idea, but then I realized that it may not work well because of the timezone differences between the client and the server. We could discuss it more, maybe there's a simple way to avoid that issue.